src/design/Pagination/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import clsx from "clsx";
+
+interface PaginationProps {
+  children: React.ReactNode;
+  align?: "center" | "left" | "right";
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  children,
+  align = "right",
+}) => (
+  <div
+    className={clsx(
+      "flex border-t border-gray-2 bg-white px-4 py-3 sm:px-6 ",
```

Comment:
Dark mode colors are missing here. Please check the storybook in dark mode too, it's very easy to see that the Pagination stands on a white background in darkmode.

Reply:
yes, I checked all again